### PR TITLE
Fix typos and grammatical errors in the README

### PR DESCRIPTION
**Pull request by cnnrde**
Spelling mistakes are corrected, grammatical mistakes ironed out, clarity should be a bit better, and should (overall) be a bit easier to read.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -7,16 +7,16 @@ The RPTPN is a specification to allow for chatting, file sharing, code sharing,
 ## Spec
 How stuff works
 
-This was inspired by the dead Replit Blog Chat protocol me and Codemonkey51 started thinking up before the entire idea of a chat within the Replit Blog's logs died down. This is my best attempt at revitalizing it.
+This was inspired by the dead Replit Blog Chat protocol Codemonkey51 and I started thinking up before the entire idea of a chat within the Replit Blog's logs died down. This is my best attempt at reviving it.
 
 ### Repl Name
-To keep everything simplified, every repl running the protocol should be named rptpn. This simplifies storing information on other peers, which just makes evertything easier and much more straight forward.
+To keep everything simplified, every repl running the protocol should be named `rptpn`. This simplifies storing information on other peers, which just makes everything easier and much more straight forward.
```
Comment (19wintersp) [inline readme.md]: "RPTPN" isn't code, I don't think it should be written as such; I'd also suggest that "RPtPN" be used in terms of capitalisation.

**Review by 19wintersp — CHANGES_REQUESTED** [review]
Looks good other than that

File: readme.md (hunk 1 above)
Comment (19wintersp) [reply readme.md]: Whoops I didn't read that correctly, that's fine, maybe note that it can be case-insensitive.

Comment (cnnrde) [reply readme.md]: Oops - pushed a commit to fix that

**Review by piemadd — APPROVED** [review]

File: readme.md (hunk 1 above)
Comment (19wintersp) [reply readme.md]: Yay

**Comment by piemadd** [comment]
poop

**Pull request merged**